tests/Valid.php

```diff
@@ -49,10 +49,10 @@ public static function config(): Config {
 			linkTargetSitelinkSiteId: 'enwiki',
 			instanceOfId: new NumericPropertyId( 'P42' ),
 			facets: new FacetConfigList(
-				new FacetConfig( new ItemId( 'Q100' ), new NumericPropertyId( 'P100' ), FacetType::BOOLEAN ),
```

Comment:
Q100, Q200, P100, P200 are similar. Nice to make values more distinct. Q100 Q200 P1 P2 is less likely to result in confusion and is easier to keep in your short term memory